CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(moveit_tutorials)
 
-add_compile_options(-std=c++14)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
`CMAKE_CXX_STANDARD` was introduced in cmake 3.1. Using it here, `cmake_minimum_required` should be increased as well!

Reply:
If we change to this way of specifying the standard, I would propose to
- (1) Do it consistently throughout MoveIt (esp. for the main repository too)
- (2) `set( CMAKE_CXX_EXTENSIONS OFF )` too

Please see [this old thread](https://github.com/ros-planning/moveit_planners/pull/73) for a discussion on the various ways to specify the standard.

Reply:
I updated the `cmake_minimum_required` and added `set( CMAKE_CXX_EXTENSIONS OFF )` here. And I agreed that it should be done consistently throughout Moveit repositories.

Reply:
Do you have the cycles to contribute this style-change all throughout the main repository too?
It would be very much appreciated... (but independent of this request)

Reply:
@v4hn you mean this https://github.com/ros-planning/moveit repository, right? Sure I can do it.